### ISSUE TITLE
Crisp: need more precision in low amount value range

### ISSUE THREAD

**Issue by unfa** [robbert-vdh/nih-plug#76]
I've started using Crisp to bring back some high frequency content especially to down-pitched sources.

However I've found I am using Amount values between 1% and 4% and I wish I could go lower yet.
![image](https://github.com/robbert-vdh/nih-plug/assets/491704/337f7e88-d38c-45d7-9af4-d66e4f81a821)

As far as I'm concerned, this control could go from 0.1% to 10% rather than from 1% to 100%.

I wonder if anyone else has the same issue?

**Comment by robbert-vdh** [comment]
The display value is rounded, but parameter's actual value isn't. So it can be set as close to zero as you want. If you hold down Shift while dragging you can drag finer, and if you Alt+Click on the field you can type in a value (don't know if that works properly in Ardour, does work in Bitwig, doesn't work in REAPER). You can also do that in your host's generic UI.

**Comment by robbert-vdh** [comment]
Also, what I often end up doing if I want a bit more control is to use the 'wet only' mode and to do the mixing in my DAW. So in Bitwig I'd set it to wet only, hit ctrl+G to group, add the dry signal back in, and then adjust the wet signal's levels and maybe add an EQ or stereo width adjustment to the wet signal. That's not as easy to do in other DAWs though.

**Comment by unfa** [comment]
Yeah, adding a separate bus just for this effect is a bit much - it'll try the Shift+Drag and Alt+Click techniques you've mentioned, thanks :)
